magenta-lib/src/main/scala/magenta/Project.scala

```diff
@@ -117,6 +117,8 @@ case object UnnamedStack extends Stack {
   override def nameOption = None
 }
 
+case class Region(name: String) extends AnyVal
```

Comment:
Mainly having a sensible `toString` method given that the region will be inside tasks. I did start off using the AWS classes but found, for example, that testing was harder as the AWS region will not so easily pattern match.